### ISSUE TITLE
tRPC `connectionParams` not working

### ISSUE THREAD

**Issue by bnussman** [cah4a/trpc-bun-adapter#9]
### Summary

Hey 👋🏼 I just wanted to say thank you for this package! Seems super valuable and I really want to use it 😄 
I'm trying to use tRPC's new [`connectionParams` ](https://trpc.io/docs/websockets#connectionParams) feature over Websocket, and I'm not sure it is working / implemeted. 

### This is my Websocket Client
```typescript
const wsClient = createWSClient({
  url: 'ws://localhost:3001/trpc',
  lazy: {
    enabled: true,
    closeMs: 0,
  },
  connectionParams() {
    const token = getAuthToken();

    return { token };
  }
});

```

### This is my context function 🥺 
```typescript
export async function createContext(data: CreateBunContextOptions) {
  const bearerToken = data.req?.headers.get('authorization')?.split(' ')[1] ?? data.info?.connectionParams?.token;

  console.log("Context Data", data.req.url, data?.info?.connectionParams)

  if (!bearerToken) {
    return {};
  }

  const session = await db.query.token.findFirst({
    where: eq(token.id, bearerToken),
    with: { user: true }
  })

  if (!session) {
    return {};
  }

  return { user: session.user, token: session };
}

export type Context = Awaited<ReturnType<typeof createContext>>;
```

It seems that connection params don't make it to the `createContext` function. The following is logged to the console when the context runs

```
Context Data http://localhost:3001/trpc?connectionParams=1 undefined
```

### This is what I see over the websocket 🥺 
![Screenshot from 2024-08-16 22-54-41](https://github.com/user-attachments/assets/302633ef-c539-4d48-9cdf-8718c933de5a)

**Comment by cah4a** [comment]
Hey @bnussman, 
Thanks for kind words!

Yes, it's not implemented, will take a look onto this feature

**Comment by cah4a** [comment]
Implemented in v1.2.0.

Feel free to reopend issue if required.
